### DISCUSSION TITLE
Invisible player and npcs

### DISCUSSION THREAD

**Discussion by Sampizaa** [Monitor221hz/Pandora-Behaviour-Engine-Plus#236] [Q&A]
I'm using mo2 and installed pandora following the instructions, it generated the animations, but when I start the game my char and all of the npcs are invisible. If I delete the output and use nemesis, in game everything is ok... What can I do?

Edit: I found an issue exactly like mine and the problem was about pandora not generating correctly because of the decimal symbol, so I changed it to '.' and launched pandora again. It worked.

**Comment by Monitor221hz** [comment]
@Sampizaa can you link where you found the issue for future reference?

**Reply by Sampizaa** [reply]
This one https://github.com/Monitor144hz/Pandora-Behaviour-Engine-Plus/issues/10
